packages/unit-testing/README.md

```diff
@@ -12,17 +12,16 @@ Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestj
 
 ## Features
 - Provides a default configuration to run unit tests with Jest in SuiteCloud projects.
-- Supports unit testing for SuiteScript 2.X files.
-- Provides SuiteScript 2.X stubs for the following SuiteScript modules:
-  - **N/http**
-  - **N/https**
-  - **N/record**
-  - **N/search**
-- Allows you to create custom stubs for SuiteScript 2.X or custom modules.
+- Supports unit testing for SuiteScript 2.x files.
+- Provides SuiteScript 2.x stubs for SuiteScript modules.
+- Allows you to create custom stubs for SuiteScript 2.x or custom modules.
 - Supports code coverage capabilities through Jest.
 
+For more information about the available SuitScript 2.x modules, see [SuiteScript 2.x Modules](https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=chapter_4220488571.html).  
+For more information about all the mockable stubs, check the CORE_STUBS list in packages\unit-testing\jest-configuration\SuiteCloudJestConfiguration.js.
```

Comment:
I think we could use a link to the file here like in the line above for SuiteScript 2.x Modules, naming it SuiteCloudJestConfiguration.js

Reply:
OK, I somehow understood I had to point to the folder. Changed.